README-legacy.md

```diff
@@ -60,64 +59,61 @@ Anti-Features:
 Bug reports and patches welcome of course!
 
 
-To build and install, use either
-  pip install noaho
-or
-  # Python 2
-  python2 setup.py install # (or ... build, and copy the .so to where you want it)
-  pip install
-or
-  # Python 3
-  python3 setup.py install # (or ... build, and copy the .so to where you want it)
-
-
-API:
-    from noaho import NoAho
-    trie = NoAho()
-'text' below applies to str and unicode in Python 2, or unicode in Python 3 (all there is)
-    trie.add(key_text, optional payload)
-    (key_start, key_end, key_value) = trie.find_short(text_to_search)
-    (key_start, key_end, key_value) = trie.find_long(text_to_search)
-    (key_start, key_end, key_value) = trie.findall_short(text_to_search)
-    (key_start, key_end, key_value) = trie.findall_long(text_to_search)
-    # keyword = text_to_search[key_start:key_end]
-    trie['keyword] = key_value
-    key_value = trie.find_long(text_to_search)
-    assert len(trie)
-    assert keyword in trie
+## API
+
+```
+from noaho import NoAho
+trie = NoAho()
+# 'text' below applies to str and unicode in Python 2, or unicode in Python 3 (all there is)
```

Comment:
Now that we don't support Python 2, I think this line can only add confusion, since `str` is unicode

Reply:
I was unsure about this, should we simply remove all mentions of Python2 then?

Reply:
Yes, please!

Reply:
Done